### DISCUSSION TITLE
Help me get started

### DISCUSSION THREAD

**Discussion by hodytone** [UkrainianBanderasCat/Digital-Logic-Sim2#19] [Q&A]
I need some help installing and configuring all the software I need to develop.
Development software, Unity SDK, GIT.

**Comment by firecerne** [comment]
Hey !
It's great that you are trying to help us in this project !

To install Unity, follow [this link](https://unity3d.com/unity/beta/2022.1.0b16). It is the version we are currently using.
I think Visual Studio Code is a great developpement software, and it comes with Unity.
If you are on Windows, follow [this link](https://gitforwindows.org/) and download the latest version to install Git.
Git can seem a little hard, so you can install [Github Desktop](https://desktop.github.com/) which is a very simple to use interface of git.
Then, you can click on [this link](x-github-client://openRepo/https://github.com/UkrainianBanderasCat/Digital-Logic-Sim2) to open the repository on github desktop and download it on your computer. Then , you can do your changes in the project and they will automatically be displayed in Github Desktop. When you are done making your changes, give them a title and a description in the bottom right corner and click "Commit to Master". Github will guide you through the steps to create a fork and a pull request.
If you need help for any of this, you can just ask here.
